src/main/java/duke/logic/commands/results/CommandResult.java

```diff
@@ -0,0 +1,9 @@
+package duke.logic.commands.results;
+
+public abstract class CommandResult {
```

Comment:
abstract classes MUST have header comments. it is like a contract between the component and the client class.
how will the client class implementer know what to expect without documentation?